test/integration/fixtures/build-withConfigTs/dts.config.ts

```diff
@@ -0,0 +1,24 @@
+import { DtsOptions, RollupOptions } from '../src';
+
+const postcss = require('rollup-plugin-postcss');
+const autoprefixer = require('autoprefixer');
+const cssnano = require('cssnano');
+
+module.exports = {
```

Comment:
can we use `import` and `export deault {...}` here? I think it's as most users are doing.

Reply:
Will get that tested shortly.